features/logging/collector.feature

```diff
@@ -9,12 +9,24 @@ Feature: collector related tests
   Scenario: All nodes logs are sent to Elasticsearch
     Given the master version == "4.1"
     Given evaluation of `cluster_logging('instance').fluentd_ready_pods.map(&:ip)` is stored in the :collector_pod_ips clipboard
+    #A workaround to https://bugzilla.redhat.com/show_bug.cgi?id=1776594
+    And I wait for the ".operations" index to appear in the ES pod with labels "es-node-master=true"
+    Given I perform the HTTP request on the ES pod with labels "es-node-master=true":
+      | relative_url | .operations.* |
+      | op           | DELETE        |
+    Then the step should succeed
+    Given I perform the HTTP request on the ES pod with labels "es-node-master=true":
+      | relative_url | project.* |
+      | op           | DELETE    |
+    Then the step should succeed
+    #Workaround end
     And I wait for the ".operations" index to appear in the ES pod with labels "es-node-master=true"
     Given I get the ".operations" logging index information from a pod with labels "es-node-master=true"
     And the expression should be true> cb.index_data['docs.count'] > "0"
+    Given I wait up to 300 seconds for the steps to pass
```

Comment:
@QiaolingTang   Test Pass. https://openshift-qe-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/job/Runner-v3/81744/.   A further question: how to quota the following steps in  one block as we have use the quota Mark Already.

Reply:
@anpingli I checked the logs, looks like the job didn't run with these changes, I'm not sure what happened here. 

> how to embrace the following steps into one block as we have use the quota Mark Already.
Not sure, let me try.

Reply:
@anpingli maybe we need to consider to update these steps: https://github.com/openshift/verification-tests/pull/899/files#diff-b2d95cc514cea4d6b51728cfc37cfbdcR77-R80